package.json

```diff
@@ -6,24 +6,20 @@
     "node": "16 || 18"
   },
   "scripts": {
-    "dev": "concurrently \"yarn start\" \"yarn start-backend\"",
-    "start": "yarn workspace app start",
-    "start-backend": "yarn workspace backend start",
-    "build": "backstage-cli repo build",
+    "start": "turbo run start",
+    "build": "turbo run build",
     "tsc": "tsc",
```

Comment:
Is this part of `build`? 

If not, is it possible to break this out so that all other plugins also have a `tsc` step, and that way we can just replace this with `turbo tsc`?

Reply:
I can look into this, the reason I never looked into this is because TS tooling can be a pain in the butt.

Reply:
I don't think its worth changing because it actually takes longer to build on average and it adds extra boilerplate to each plugin. 

Here are my averages after some testing: 
| Before | After  |
| ------- | ------ |
| 11.5s   | 13.1s  |

Reply:
Okay, that's fine, we can just leave it as is then.